### PR TITLE
WE-451 Added endpoint in API Routes for wbGeometry

### PR DESCRIPTION
**Pull request by stokkoff**
# Request Template Witsml Explorer
Thank you for contributing to WITSML Explorer!
Before submitting this PR, please fill in the following template.

## Fixes
This pull request fixes #WE-451

## Description
_Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change._

No dependencies and the descubtion is the same as the title

## Type of change
_List the types of change. Remove from the list any points which are not necessary_

* New feature (non-breaking change which adds functionality)

## Impacted Areas in Application
_List general components of the application that this PR will affect:_

*This PR will not change anything as it is only backend

## Checklist:
_Please tick all the boxes or remove the ones that aren't needed and explain why_

*Communication*
* [x] PR is related to an issue
* [x] I have made corresponding changes to the documentation

*Code quality*
* [x] Code follows the style guidelines
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] Existing tests pass
* [x] New code is covered by passing tests

## Further comments
_If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc..._

### CODE REVIEW

File: Src/WitsmlExplorer.Api/Models/WbGeometry.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        public string UidWell { get; set; }
+        public string UidWellbore { get; set; }
+        public string Uid { get; set; }
+        public string NameWell { get; set; }
+        public string NameWellbore { get; set; }
+        public DateTime? DTimReport { get; set; }
+        public MeasuredDepthCoord MdBottom { get; set; }
+        public LengthMeasure GapAir { get; set; }
+        public LengthMeasure DepthWaterMean { get; set; }
+        public string SourceName { get; set; }
```
Comment (janburak) [inline Src/WitsmlExplorer.Api/Models/WbGeometry.cs]: The last five fields should be instead a CommonData class

File: Src/Witsml/Data/Measures/WitsmlMeasuredDepthCoord.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 namespace Witsml.Data.Measures
 {
-    public class WitsmlMeasuredDepthCoord : MeasureWithDatum
+    public class WitsmlMeasuredDepthCoord : Measure
```
Comment (janburak) [inline Src/Witsml/Data/Measures/WitsmlMeasuredDepthCoord.cs]: Why is this changed?

File: Src/WitsmlExplorer.Api/Services/WbGeometryService.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                    MdBottom = (wbGeometry.MdBottom == null) ? null : new MeasuredDepthCoord { Uom = wbGeometry.MdBottom.Uom, Value = double.Parse(wbGeometry.MdBottom.Value, CultureInfo.InvariantCulture) },
+                    GapAir = (wbGeometry.GapAir == null) ? null : new LengthMeasure { Uom = wbGeometry.GapAir.Uom, Value = decimal.Parse(wbGeometry.GapAir.Value) },
+                    DepthWaterMean = (wbGeometry.DepthWaterMean == null) ? null : new LengthMeasure { Uom = wbGeometry.DepthWaterMean.Uom, Value = decimal.Parse(wbGeometry.DepthWaterMean.Value) },
+                    SourceName = wbGeometry.CommonData.SourceName,
+                    ItemState = wbGeometry.CommonData.ItemState,
+                    Comments = wbGeometry.CommonData.Comments,
+                    DTimCreation = StringHelpers.ToDateTime(wbGeometry.CommonData.DTimCreation),
+                    DTimLastChange = StringHelpers.ToDateTime(wbGeometry.CommonData.DTimLastChange),
+
+                }).OrderBy(risk => risk.DTimReport);
```
Comment (janburak) [inline Src/WitsmlExplorer.Api/Services/WbGeometryService.cs]: A bit wrong to call this "risk" instead of "wbGeometry"

**Comment by janburak** [comment]
"g" should be capitalized in all of the three "wbgeomtry(s)" strings in WitsmlWbGeometrys.cs

**Comment by janburak** [comment]
The "Fixes" sections erroneously links to "#451" from GitHub instead of "WE-451" from Jira

File: Src/Witsml/Data/Measures/WitsmlMeasuredDepthCoord.cs (hunk 2 above)
Comment (stokkoff) [reply Src/Witsml/Data/Measures/WitsmlMeasuredDepthCoord.cs]: Was an issue with adding a Value to the Meeasured DepthCoord but it is fixed now with the changes Eirin made to the file so I'll revert this change

Comment (stokkoff) [reply Src/Witsml/Data/Measures/WitsmlMeasuredDepthCoord.cs:3]: I dont't know how to fix this, tried finding the issue WE-451 but it did not appear

**Review by janburak — APPROVED** [review]
- [x] Read issue and PR
- [x] Pulled branch and manually tested
- [x] Verify that PR resolves issue
- [x] Reviewed the code

**Review by steinsiv — APPROVED** [review]
- [x] Read issue and PR
- [x] Pulled branch and manually tested
- [x] Verify that PR resolves issue
- [x] Reviewed the code
